ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -483,14 +491,16 @@ object BigDiffy extends Command {
 
   /** Scio pipeline for BigDiffy. */
   //scalastyle:off cyclomatic.complexity
+  //scalastyle:off method.length
```

Comment:
nit: it's common to put all scalastyle configs on one line like //scalastyle:off cyclomatic.complexity method.length